### DISCUSSION TITLE
Welcome to tortoise-tts-fastest Discussions!

### DISCUSSION THREAD

**Discussion by manmay-nakhashi** [manmay-nakhashi/tortoise-tts-fastest#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bonk1971** [comment]
Thank you for your work on this.  I found you from the link on the Tortoise-TTS discussion announcing that you will be helping maintain that repo.  Will your changes here be incorporated there or will this remain a separate repo?

**Reply by manmay-nakhashi** [reply]
@Bonk1971 this will remain a separate repo , but i'll update this changes to the main repo , i'll be updating kv_cache , but i won't change much , because main repo api needs to be backword compatible, if there is more improvement i'll first try it here , once it is stable i'll update in main repo.

**Comment by Bonk1971** [comment]
Can you tell me what this means?  "AMD INSTALLATION IS NOT SUPPORTED, please don't try it" Is this referring to AMD processors or something else.
